src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -769,6 +770,11 @@ public string OutputResultsCacheFile
         /// </summary>
         public bool DiscardBuildResults { get; set; } = false;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the build process should run as low priority.
+        /// </summary>
+        public bool LowPriority { get; set; }
```

Comment:
Would you mind providing more details on when something needs to survive serialization?

Priority passes automatically from parent process to child process, and it's also passed explicitly as a parameter at node creation. (Technically, we could get it from the process and avoid passing that around, but it would've made it harder if we had taken the high priority node creation change.) Is that sufficient to say it doesn't need to be translated or are there other necessary conditions?